src/main/java/baseball/controller/Game.java

```diff
@@ -0,0 +1,72 @@
+package baseball.controller;
+
+import baseball.model.Answer;
+import baseball.view.InputView;
+import baseball.view.ResultView;
+
+public class Game {
+
+    private final Answer answer = new Answer();
+    private final InputView input = new InputView();
+    private final ResultView output = new ResultView();
+    private boolean isRunning = true;
+
+    public void run() {
+        while (isRunning) {
+            System.out.println(answer); // 테스트용
+            String userGuess = input.getUserGuess();
+            compareWithAnswer(userGuess);
+        }
+    }
```

Comment:
전역변수 사용을 최대한 안하고 싶은데
```suggestion

    public void run() {
        while (true) {
            System.out.println(answer); // 테스트용
            String userGuess = input.getUserGuess();
            compareWithAnswer(userGuess);
        }
    }
```

이렇게 고치고, 아래 `terminate()` 메서드 내에서 `System.exit(0);`을 호출하는 방법으로 개선할 수 있을거같은데
소나린트에서 아주 노란줄이 찍찍 난리가 나서... 이렇게 했는데요 뭐가 더 나은 방법일까요..?